mobile/android/android-livesync-service.ts

```diff
@@ -0,0 +1,63 @@
+///<reference path="../../../.d.ts"/>
+"use strict";
+
+import helpers = require("../../helpers");
+import path = require("path");
+import temp = require("temp");
+import util = require("util");
+
+class LiveSyncCommands {
+	public static DeployProjectCommand(liveSyncUrl: string): string {
+		return util.format("DeployProject %s \r", liveSyncUrl);
```

Comment:
You can use template strings and remove util require. Also what's the idea of `\r`?